driver/configuration/provider_viper.go

```diff
@@ -48,6 +49,7 @@ const (
 	ViperKeyPublicListenOnHost             = "serve.public.host"
 	ViperKeyPublicListenOnPort             = "serve.public.port"
 	ViperKeyPublicDisableHealthAccessLog   = "serve.public.access_log.disable_for_health"
+	ViperKeyCookieSameSiteMode             = "serve.cookie_same_site_mode"
```

Comment:
Let's actually move this one key down: `serve.cookies.same_site_mode` :)

```suggestion
	ViperKeyCookieSameSiteMode             = "serve.cookies.same_site_mode"
```